index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <main id="main">
+      <div id="title">
+        <h1>Fahim Morshed</h1>
+      </div>
+      <div id="img-div">
+        <img id="image" />
```

Comment:
An `img` tag has been used but not utilized. It would be great importing a simple image using `src` attribute inside the image element to make the site looks better.

Reply:
Hey great job 💯  on using inline comments but if you want to help the student more you should mention why it's better to use `src` and provide some documentations 😉

Reply:
An `img` tag has been used but not utilized. It would be great importing a simple image using `src` attribute inside the image element to make the site looks better. A The `src` attribute specifies the URL of the media file to play. To get more detail information you can visit this [URL](https://www.w3schools.com/tags/att_source_src.asp#:~:text=Definition%20and%20Usage,audio%3E%20and%20.).